console/console-init/ui/src/modules/msg-and-iot/utils/ProjectFormatter.tsx

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import {
+  CheckCircleIcon,
+  InProgressIcon,
+  ErrorCircleOIcon,
+  PendingIcon
+} from "@patternfly/react-icons";
+import { Flex, FlexItem } from "@patternfly/react-core";
+import { kFormatter } from "utils";
+import { StyleSheet, css } from "@patternfly/react-styles";
+import { ProjectTypes } from "./constant";
+
+const style = StyleSheet.create({
+  red_color: {
+    color: "var(--pf-global--palette--red-100)"
+  }
+});
+interface IProjectTypePlanProps {
+  type: string;
+  plan?: string;
+}
+
+interface IProjectStatusProps {
+  phase: string;
+  messages?: Array<string>;
+}
+interface IProjectErrorProps {
+  errorCount?: number;
+  errorMessages?: Array<string>;
+}
+interface IProjectEntitiesProps {
+  projectType: ProjectTypes;
+  addressCount?: number;
+  connectionCount?: number;
+  deviceCount?: number;
+  activeCount?: number;
+}
+interface IProjectErrorMessagesProps {
+  messages?: Array<string>;
+}
+
+const statusToDisplay = (phase: string) => {
```

Comment:
can revisit the function name statusToDisplay.
This function purpose to return icon based on phase. I would like to suggest one name as
getStatusIconByPhase